kevm-pyk/src/kevm_pyk/kproj/evm-semantics/driver.md

```diff
@@ -122,9 +123,9 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
            <nonce> NONCE </nonce>
            ...
          </account>
-         <accessedAccounts> _ => #if Ghaswarmcoinbase << SCHED >> #then SetItem(MINER) #else .Set #fi </accessedAccounts>
+         <accessedAccounts> _ => #if Ghaswarmcoinbase(SCHEDT) #then SetItem(MINER) #else .Set #fi </accessedAccounts>
```

Comment:
Where are getters like `Ghaswarmcoinbase(ScheduleTuple)` defined?

Reply:
here: https://github.com/runtimeverification/evm-semantics/blob/09234061c9cc1f72ffa13b02193c3970258ceeff/kevm-pyk/src/kevm_pyk/kproj/evm-semantics/schedule.md?plain=1#L59

They are generated by the compiler for each named nonterminal.